internal/sub_reconcilers/reconcile_deployment.go

```diff
@@ -61,18 +64,44 @@ func (r *DeploymentReconciler) Reconcile(ctx context.Context, bestie *petsv1.Bes
 	if err != nil && errors.IsNotFound(err) {
 		log.Info("Creating a new app for bestie")
 		fileName := "config/resources/bestie-deploy.yaml"
-		err := r.applyManifests(ctx, bestie, dp, fileName)
+		err := r.applyDeploymentFile(ctx, bestie, *dp.DeepCopy(), fileName)
 		if err != nil {
 			return ctrl.Result{}, fmt.Errorf("Error during Manifests apply - %w", err)
 		}
 	}
 	return ctrl.Result{}, err
 }
 
-func (r *DeploymentReconciler) applyManifests(ctx context.Context, bestie *petsv1.Bestie, obj client.Object, fileName string) error {
-	err := util.ApplyManifests(r.client, r.Scheme, ctx, bestie, obj, fileName)
+func (r *DeploymentReconciler) applyDeploymentFile(ctx context.Context, bestie *petsv1.Bestie, obj appsv1.Deployment, fileName string) error {
```

Comment:
Could you please write a comment describing what this function does?

Reply:
the function name is exactly what it does ?